VKApp/VKApp/Services/NetworkService.swift

```diff
@@ -49,6 +50,26 @@ final class NetworkService {
         }
     }
 
+    func fetchUsersPromise(_ method: String) -> Promise<[UserItem]> {
+        var parametrs: Parameters = [
+            Constants.accessTokenText: Session.shared.token,
+            Constants.vText: Constants.apiVersionText,
+            Constants.fieldsParametrName: Constants.getPhotoParametrName,
```

Comment:
```suggestion
            Constants.fieldsParameterName: Constants.getPhotoParameterName,
```
и дальше поправь ошибки в слове